test/over_react/component_declaration/transformer_integration_tests/component_integration_test.dart

```diff
@@ -22,12 +22,6 @@ import '../../../test_util/test_util.dart';
 
 main() {
   group('transformed component integration:', () {
-    test('props class cannot be instantiated directly', () {
```

Comment:
Prop classes are made abstract by the transformer and the DDC was failing on instantiating an abstract object.